content/user/customizing/add_pages.md

```diff
@@ -51,3 +51,14 @@ Once you have created your new page, you'll want to link to it. A few options ar
 - If you want the link in the Information Sidebox, use the instructions on [adding a link to the Information Sidebox](/user/sideboxes/add_link_information_sidebox/).  
 - You can put the link in the [header](/user/template/header/) or [footer](/user/template/footer/) of your site, but the method of doing so will depend on your template.
 
+## Direct Access to Files 
+
+You may wish to simply provide files (such as PDFs) on your site without requiring the registration and purchase steps associated with paid downloads.
+
+The recommended method of doing this is as follows:
+
+- Create a new folder at the top level of your cart.  Remember that Zen Cart already comes with a `/docs` folder, so choose another name. 
+- Prior to populating the folder, create an `index.html` and `.htaccess` file that limits access in the folder.  You can copy these files from the `/images` folder and customize as needed.  The images folder doesn't permit the use of PDF files, so you'll have to add that to the `.htaccess` file. 
+
+Note: If you are using Ngnix or another webserver other than Apache, you will have to make the approriate security changes for that system. 
```

Comment:
Since this "Note:" goes with the prior line, maybe skip the line-break so that it shows in the same context, since it's part of the same step?